extensions/nestcamera/README.md

```diff
@@ -0,0 +1,217 @@
+# Nest Camera Raycast Extension
+
+A Raycast extension that provides quick access to your Google Nest camera feeds through direct RTSP streaming. View your cameras instantly with FFplay for low-latency, high-quality playback. This extension is specifically designed for the Nest Cam Indoor (1st gen) model.
+
+## Camera Compatibility
+
+This extension is built specifically for:
+- **Nest Cam Indoor (1st generation)**
+  - Only supports RTSP streaming
+  - Does not support newer protocols like WebRTC
+  - Requires FFplay for direct RTSP playback
+
+Other Nest camera models are not currently supported as they use different streaming protocols.
+
+## Features
+
+- 🎥 Instant access to Nest camera feeds through direct RTSP streaming
+- 🚀 Low-latency playback with optimized FFplay parameters
+- 🔍 Quick search and filter through your cameras
+- ⭐️ Favorite cameras for hotkey access
+- 🎮 One-click stream launch
+- 🔐 Secure OAuth2 authentication with PKCE
+- 🧹 Automatic process management and cleanup
+- ⚡️ Quick Access to your preferred camera
+
+## Prerequisites
+
+1. Nest Cam Indoor (1st generation)
+2. macOS 12.0 or later
+3. Raycast 1.50.0 or later
+4. **FFmpeg/FFplay installed** - This is **required** for video playback
+5. Google Device Access registration ($5 one-time fee)
+6. Google Cloud OAuth2 client credentials
+
+### Required Software
+
+The extension relies on the following software being installed on your Mac:
+- **FFmpeg/FFplay**: Used for RTSP video streaming
+- **AppleScript**: Used for window management and loading dialogs (built into macOS)
+- **Bash**: Used for scripting (built into macOS)
+
+### Installing FFmpeg
+
+FFmpeg is required for this extension to work. You can install it using Homebrew:
+
+```bash
+# Install Homebrew if you don't have it
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+# Install FFmpeg
+brew install ffmpeg
+```
+
+Alternatively, you can download FFmpeg from the [official website](https://ffmpeg.org/download.html).
+
+If you encounter any issues with the extension, please ensure these prerequisites are properly installed.
+
+## Installation
+
+1. Install FFmpeg/FFplay (if not already installed):
+   ```bash
+   brew install ffmpeg
+   ```
+
+2. Install the extension from Raycast Store
+
+3. Set up Google Device Access:
+   - Visit [Google Device Access Console](https://console.nest.google.com/device-access)
+   - Complete registration ($5 one-time fee)
+   - Create a project and note the Project ID
+
+4. Configure OAuth2:
```

Comment:
I tried to set this up but I keep getting an error

![Raycast 2025-03-06 at 09 53 53](https://github.com/user-attachments/assets/5da2ad91-fb4b-4777-96c8-bea4a87610aa)

I tried to follow all steps, something like OAuth client ID application type could be nice